l1-contracts/contracts/zksync/interfaces/IExecutor.sol

```diff
@@ -25,6 +44,16 @@ uint256 constant L2_LOG_KEY_OFFSET = 24;
 /// @dev Offset used to pull Value From Log. Equal to 4 (bytes for isService) + 20 (bytes for address) + 32 (bytes for key)
 uint256 constant L2_LOG_VALUE_OFFSET = 56;
 
+/// @dev BLS Modulus value defined in EIP-4844 and the magic value returned from a successful call to the
+/// point evaluation precompile
+uint256 constant BLS_MODULUS = 52435875175126190479447740508185965837690552500527637822603658699938581184513;
+
+/// @dev Total size of the input for point evaluation precompile without the version hash
+uint256 constant POINT_EVALUATION_INPUT_SIZE = 160;
+
+/// @dev Packed pubdata commitments. ∑ claimed value (32 bytes) || commitment (48 bytes) || proof (48 bytes)) = 128 bytes
```

Comment:
nit & optional, I would personally prefer not to use special symbols, but if you think it is much more beautiful this way, then ok

Reply:
yea we can get rid of it, i thought it looked clean but i dont have a strong preference